provider/cpu_utils.py

```diff
@@ -111,7 +111,7 @@ def check_cpu_flags(params, flags, test, session=None):
     :param test: QEMU test object
     :param session: guest session
     """
-    cmd = params["check_flag_cmd"]
+    cmd = "lscpu | grep Flags | awk -F ':'  '{print $2}'"
```

Comment:
I will prefer `params.get("check_flag_cmd", "lscpu | grep Flags | awk -F ':'  '{print $2}'")`

Reply:
LGTM otherwise, and request test result for latest version. Thanks.

Reply:
There is unlikely another cmd we will use to check flags. I'd like to keep it is and update in the future if it's not suitable.
Latest test results are as below, the failed case is due to a qemu bug.
 (1/4) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.smt_test.with_EPYC: PASS (119.12 s)
 (2/4) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.smt_test.with_EPYC.no_topoext: PASS (118.87 s)
 (3/4) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.smt_test.with_Opteron: PASS (121.33 s)
 (4/4) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.smt_test.with_Opteron.with_topoext: FAIL: Guest cpu threads number is not right (36.72 s)
 (1/1) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.x86_cpu_model.EPYC: PASS (86.20 s)
Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.amd.flag_1: PASS (49.90 s)